### PR TITLE
Describe governance model

### PR DESCRIPTION
**Pull request by pkosiec**
## Description

Changes proposed in this pull request:

- Describe governance model

As this is the repository where we keep the community health files, we agreed to not create `community` repo for now and keep the `GOVERNANCE.md` file here.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+### Support
+
+All participants in the community are encouraged to provide support for new users within the project management infrastructure. This support is provided as a way of growing the community. Those seeking support should recognise that all support activity within the project is voluntary and is therefore provided as and when time allows. A user requiring guaranteed response times or results should therefore seek to purchase a support contract from a vendor. (Of course, that vendor should be an active member of the community.) However, for those willing to engage with the project on its own terms, and willing to help support other users, the community support channels are ideal.
+
+### Contribution Process
+
+Anyone can contribute to the project, regardless of their skills, as there are many ways to contribute. For instance, a contributor might be active on the project mailing list and issue tracker, or might supply patches. The various ways of contributing are described in more detail in our roles in open source document.
+
+The developer mailing list is the most appropriate place for a contributor to ask for help when making their first contribution.
```
Comment (mszostok) [inline GOVERNANCE.md:84]:
```suggestion
The developer mailing list is the most appropriate place for a contributor to ask for help when making their first contribution.
```
Maybe in our case we can "redirect" them to Slack channel?
Probably we will set up a mailing list too, but Slack is more convenient, at least in my opinion.

**Review by mszostok — APPROVED** [review]
Overall it LGTM, just one suggestion. 

Probably we should wait also for Paul to approve it too.

File: GOVERNANCE.md (hunk 1 above)
Comment (pkosiec) [reply GOVERNANCE.md]: Good idea - I updated the document.

**Review by platten — APPROVED** [review]
LGTM

**Pull request merged**